packages/pwa-kit-react-sdk/src/ssr/universal/utils.js

```diff
@@ -11,6 +11,8 @@ import {proxyConfigs} from '../../utils/ssr-shared'
 
 const onClient = typeof window !== 'undefined'
 
+let _config
```

Comment:
Do not do this, this will leak on configuration for all clients making requests to the lambda. Normally this would be desirable because there is a 1-1 mapping of a lambda to a configuration. But that only holds true when using the default logic for loading your config, it breaks if you were to customize the config file resolving to base your choice from the url path for example.